### ISSUE TITLE
Tabbedpanel Header is scrollable when using touch screen

### ISSUE THREAD

**Issue by Gobbel2000** [kivy/kivy#7131]
**Software Versions**
* Python: 3.7.3
* OS: Raspberrypi OS on Raspberrypi 4
* Kivy: v2.0.0rc3 and git master
* Kivy installation method: pip and from source respectively

**Describe the bug**
When dragging a horizontal tabbedpanel header up and down on a touch screen it scrolls with the touch. This did not happen when using a mouse for input. The header buttons are all visible simultaneously, see notes for reproduction. 

**Expected behavior**
The header buttons should not move when dragged, as long as all of them fit in frame.

**To Reproduce**
The file at **examples/widgets/tabbedpanel.py** is enough to reproduce the issue, although it has to be run with touch screen input. I used the configuration as set in the code below, other than that this is just the tabbedpanel.py file. I'm using the **libinput** touch input driver. 

**Code and Logs and screenshots**
```python
'''
TabbedPanel
============

Test of the widget TabbedPanel.
'''
from kivy.app import App
from kivy.config import Config
from kivy.uix.tabbedpanel import TabbedPanel
from kivy.lang import Builder

Config.set("input", "%(name)s", "probesysfs,provider=hidinput")
Config.set("input", "device_%(name)s",
           "probesysfs,provider=mtdev,param=rotation=0,param=invert_y=0")

Builder.load_string("""

<Test>:
    size_hint: .5, .5
    pos_hint: {'center_x': .5, 'center_y': .5}
    do_default_tab: False

    TabbedPanelItem:
        text: 'first tab'
        Label:
            text: 'First tab content area'
    TabbedPanelItem:
        text: 'tab2'
        BoxLayout:
            Label:
                text: 'Second tab content area'
            Button:
                text: 'Button that does nothing'
    TabbedPanelItem:
        text: 'tab3'
        RstDocument:
            text:
                '\\n'.join(("Hello world", "-----------",
                "You are in the third tab."))

""")


class Test(TabbedPanel):
    pass


class TabbedPanelApp(App):
    def build(self):
        return Test()


if __name__ == '__main__':
    TabbedPanelApp().run()
```

**Comment by Gobbel2000** [comment]
This is fixed by #7129.

**Comment by Zen-CODE** [comment]
Seems like this is fixed than? Please re-open if not. Thanks
